pkg/printers/internalversion/printers.go

```diff
@@ -1154,11 +1154,24 @@ func formatHosts(rules []networking.IngressRule) string {
 	return ret
 }
 
-func formatPorts(tls []networking.IngressTLS) string {
-	if len(tls) != 0 {
-		return "80, 443"
+func formatPorts(obj *networking.Ingress) string {
+	allowHTTP := true
```

Comment:
All lgtm, but for my understanding `kubectl get` converts info from server, seems not very good to displaying such info in hardcode way. https://github.com/kubernetes/kubernetes/pull/26684

Reply:
Agreed that hard coding the port is strange, but it is currently hard coded and there isn't currently any way to specify a different port for ingress HTTP or HTTPS, so that it beyond the scope of this PR to change that.